### PR TITLE
Pre check vclist from yarn and remove the vc type group which is not in.

### PR DESCRIPTION
**Pull request by ydye**

### CODE REVIEW

File: src/rest-server/src/models/v2/group.js [hunk 1]
```diff
@@ -412,15 +413,34 @@ const updateUserGroupAndVirtualCluster = async () => {
...
+      }
+    }
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.log(error);
+  }
+};
+
 if (config.env !== 'test') {
   (async function() {
```
Comment (debuggy) [inline src/rest-server/src/models/v2/group.js]: why wrap it with async function

Comment (ydye) [reply src/rest-server/src/models/v2/group.js]: All function is async, we should await it.

Comment (debuggy) [reply src/rest-server/src/models/v2/group.js]: however the groupTypeVCCheck is already async function, maybe you could use await directly

Comment (ydye) [reply src/rest-server/src/models/v2/group.js]: https://stackoverflow.com/questions/39679505/using-await-outside-of-an-async-function

**Pull request merged**